### DISCUSSION TITLE
Electing this project to be the one listed in fsprojects or extending maintainers to the old one

### DISCUSSION THREAD

**Discussion by Esgariot** [faldor20/fsharp-language-server#15] [Ideas]
Hi, I've stumbled upon this repo only by going through issues in [fsprojects/fsharp-language-server](https://github.com/fsprojects/fsharp-language-server) and given the better maintenance status and poor discoverability I'd like this one to merge/replace the one mentioned. We could write an email to [fsprojects@fastmail.fm](mailto:fsprojects@fastmail.fm) linking this repo or something.

**Comment by faldor20** [comment]
I'd be happy to do that, I was a little apathetic about it because, I was under the impression the whole point of joining their group is to stop things like this from happening, but I might be misguided about that.
Either way, I have made a request.
https://github.com/fsprojects/fsprojects-admin/issues/40

**Comment by faldor20** [comment]
I now have access ti the official fsharp language sever repo and will be moving all development over to there.
Thanks for the prompting @Esgariot :)
